modules/cbProcessFlow/pushAlongFlow.php

```diff
@@ -47,12 +47,17 @@ public function process($context = false) {
 			where deleted=0 and cbprocessflowid=?',
 			array($pflowid)
 		);
+		$shownodata = $this->getFromContext('shownodata');
 		if (!$rs || $adb->num_rows($rs)==0) {
 			return getTranslatedString('LBL_NO_DATA');
```

Comment:
do this one too, please

Reply:
@joebordes - I already fixed it, please have a look.